uiscene_patch.py

```
@@ -0,0 +1,73 @@
+                        onComplete: () => {
+                            this.graphics.setVisible(false);
+                            this.splashScreen.setVisible(false);
+                        }
+                    });
+                });
+            }
+        });
+    }"""
+
+    content = content.replace(old_anim, new_anim)
+    with open(path, 'w') as f: f.write(content)
```

Ruff (0.15.2)

[error] Multiple statements on one line (colon)

(E701)